web/src/components/atoms/NotFound/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { useIntl } from "react-intl";
```

Comment:
Can you introduce react-i18nnext instead of react-intl?

Reply:
I removed `react-intl` for now.
Will add `react-i18nnext` it in another issue